conjure-java-core/build.gradle

```diff
@@ -38,13 +38,15 @@ dependencies {
     compile 'com.google.googlejavaformat:google-java-format'
     compile 'com.google.guava:guava'
     compile 'com.palantir.conjure.java.api:errors'
+    compile 'com.palantir.dialogue:dialogue-client'
```

Comment:
Do we want to make this repo public first?

Reply:
done